src/shared/containers/AddWorkspaceContainer.tsx

```diff
@@ -60,6 +60,7 @@ const AddWorkspaceContainer: React.FC<{
           workspaces: updatedWorkspacesList(newWorkspaceId),
           label: studio.label,
           description: studio.description,
+          type: studio['@type'],
```

Comment:
please destructure the entire studio there, instead of deliberately adding each key